php/php.editor/src/org/netbeans/modules/php/editor/completion/CompletionContextFinder.java

```diff
@@ -228,6 +238,36 @@ final class CompletionContextFinder {
             new Object[]{PHPTokenId.PHP_VARIABLE, PHPTokenId.PHP_TOKEN});
     private static final List<String> SERVER_ARRAY_TOKENTEXTS =
             Arrays.asList(new String[]{"$_SERVER", "["}); //NOI18N
+   private static final Set<PHPTokenId> BUILT_IN_TYPES = new HashSet<>(Arrays.asList(
```

Comment:
Cannot we use `Set.of()`? That would also make the fields immutable, so definitely thread-safe.

Reply:
Yes, we should use it. PHP editor is `javac.source=1.8`. We should change it to `javac.release=17`. OK, I'll try changing it. Thanks!

Reply:
Perfect! Thanks!